drivers/usb/device/usb_dc_mcux.c

```diff
@@ -268,8 +274,10 @@ int usb_dc_ep_configure(const struct usb_dc_ep_cfg_data *const cfg)
 		LOG_WRN("Failed to un-initialize endpoint (status=%d)", (int)status);
 	}
 
+#ifdef FSL_FEATURE_USBHSD_USB_RAM_BASE_ADDRESS
 	/* Allocate buffers used during read operation */
```

Comment:
Minor question- should this comment state "Allocate buffers used during read operation"? Based on my understanding of USB RAM, it seems it is only used by the USB write operation.

Reply:
out direction is relative to the host, as this is a driver for a device, "out" would be a read